### ISSUE TITLE
The list of <p:selectOneMenu> position is strange, when submit with "@form" update.

### ISSUE THREAD

**Issue by hir0ku** [adminfaces/admin-template#190]
##### Issue Overview
when use "selectOneMenu" of PrimeFaces component, list position is strange.
![selectOneMenuListPosition](https://user-images.githubusercontent.com/57468638/68356170-ea6b4280-0154-11ea-8076-e08662b84a0e.png)

##### Current Behaviour

- <p:selectOneMenu> list position is header of page, when "commandButton" perfomed with "@form" update.

##### Expected Behaviour

- <p:selectOneMenu> list position expect under <p:selectOneMenu> component, when "commandButton" perfomed with "@form" update.

##### How to reproduce  

```xhtml
<p:panelGrid id="panelGrid">
    <p:outputLabel for="@next" value="select one"/>
    <p:selectOneMenu value="#{indexView.selectOne}">
        <f:selectItems value="#{indexView.itemCandidates}" var="i" itemLabel="#{i}" itemValue="#{i}"/>
    </p:selectOneMenu>

    <p:commandButton value="submit"
                     process="panelGrid"
                     update="@form"
                     actionListener="#{indexView.submit}"/>

</p:panelGrid>
```

entire code is here. https://github.com/hir0ku/adminfaces-operation-verification/blob/master/src/main/webapp/index.xhtml

##### Additional Information

* AdminFaces version:  1.0.0
  * also try 1.0.2, same problem.
* PrimeFaces version: 7.0
* JSF implementation: Wildfly 16.0.0.Final Mojarra 2.3.9.SP01

**Comment by hir0ku** [comment]
I remove "org.omnifaces.resourcehandler.CombinedResourceHandler" from faces-config.xml.
This problem dosen't occur.

**Comment by rmpestano** [comment]
Hi @hir0ku, [showcase](https://github.com/adminfaces/admin-showcase/blob/master/src/main/webapp/WEB-INF/faces-config.xml#L7) and starter projects are using it so it should work together.

Can you reproduce here? https://adminfaces-adminfaces.apps.ca-central-1.starter.openshift-online.com/pages/sample/sample-form.xhtml? it uses selectCheckBoxMenu but would be easy to replace with selectOne/ManyMenu. If I find some time I can try that.

Also make sure to have clean browser caches

**Comment by hir0ku** [comment]
@rmpestano Thank you for your comments and giving hints and separating your time.
I tried the showcase application in my enviroment, but this problem dosen't occur... 😢 
I will continue to try to find the cause...

**Comment by rmpestano** [comment]
I think I made it work in your project, just upgrade OmniFaces to `v3.2`:

```
<dependency>
            <groupId>com.github.adminfaces</groupId>
            <artifactId>admin-template</artifactId>
            <version>${adminfaces.version}</version>
        </dependency>
        <dependency>
            <groupId>org.omnifaces</groupId>
            <artifactId>omnifaces</artifactId>
            <version>3.2</version>
        </dependency>
```

Can you try that?

**Comment by hir0ku** [comment]
It's work! 
Thank you very much!

**Comment by rmpestano** [comment]
Hi, 

> I think, admin-template should depends on 3.2 version of omnifaces.

The problem is that OmniFaces 3.2 is not compatible with JavaEE 6 (JSF 2.1, CDI 1.0 etc...) so we may break some apps if do the upgrade.

I think in admin-template 2.x we could make OmniFaces 3.x default as JavaEE6 is is 10 years old already.

**Comment by rmpestano** [comment]
Also it required Java 8:

> OmniFaces 3.0 will explicitly prevent deployment when Java 1.8, JSF 2.3 or CDI 1.1 are absent in the environment.

**Comment by hir0ku** [comment]
I understand the circumstances surrounding this library.
This issue will be closing.
Thank you very much.

**Comment by rmpestano** [comment]
I've made some tests and your issue is also fixed with OmniFaces `2.7.1` so I'll upgrade admin-template to this version as it doesn't conflict with JavaEE 6 nor older versions of Java.

It will be available in admin-template `1.1.0`.

Thank you very much for the feedback!
